documentation/experimental_features.md

```diff
@@ -6,7 +6,136 @@ while iterating on new functionality. Almost all experimental features are
 eventually stabilized in future releases. While a feature is experimental, its
 API might change, requiring the user to update their code or configuration. The
 Bolt team attempts to make these changes painless by providing useful warnings
-around breaking behavior where possible. 
+around breaking behavior where possible.
+
+## Configuration policies and the policy command
+
+This feature was introduced in [Bolt 3.21.0](). 
+
+### What are policies?
+
+Configuration policies, or policies for short, are public classes that can be
+applied directly to one or more targets. Policies are Puppet code stored in the
+`manifests/` directory of modules on the modulepath. They can be applied
+directly to targets just like other Puppet code.
+
+### Policies in a Bolt project's configuration file
+
+The project configuration file, `bolt-project.yaml`, supports a `policies` key
+which lists the policies available to a project. A Puppet class becomes a policy
+when it's listed under the `policies` key, and if the `policies` key is not
+configured then no policies will be available for Bolt to apply.
+
+Policies may be manually added to the `policies` key in `bolt-project.yaml` or
+managed via Bolt commands. This key supports glob pattern for easily listing
+multiple policies, i.e. boltproject::* includes boltproject::admin and
+boltproject::sshkeys policies.
+
+The following configuration file makes the `bolt::project::admin` and
+`bolt::project::sshkeys` policies available to the project:
+```
+---
+name: bolt
+policies:
+- bolt::project::admin
+- bolt::project::sshkeys
+```
+
+### Creating new policies 
+
+You can use Bolt to create a new policy in your project and add it to the
+`policies` key in your project configuration. For example, to create a new
+project-level policy named `bolt::project::user`, run:
+
+* _*nix shell command_
+  ```
+  bolt policy new bolt::project::user
+  ```
+
+* _PowerShell cmdlet_
+  ```
+  New-BoltPolicy bolt::project::user
+  ```
+
+An empty class is created in the project's `manifests/` directory that you can
+then populate.
+
+Example output:
+```
+Created policy 'bolt::project::user' at '/Users/puppet.user/bolt/manifests/project/user.pp'
+```
+> Policy names must follow [class naming
+conventions](https://puppet.com/docs/puppet/7/lang_reserved.html#classes-and-defined-resource-type-names).
```

Comment:
```suggestion
Policy names must follow [class naming
conventions](https://puppet.com/docs/puppet/7/lang_reserved.html#classes-and-defined-resource-type-names).
```

Hm, I think this might work better as the first line at line 60.